### ISSUE TITLE
AdbCommands instance as first argument

### ISSUE THREAD

**Issue by badgerhome** [happyleavesaoc/python-firetv#57]
(firetv) firetv@debian:/srv$ firetv-server -d 192.168.1.108:5555
Traceback (most recent call last):
  File "/srv/firetv/firetv/bin/firetv-server", line 11, in <module>
    sys.exit(main())
  File "/srv/firetv/firetv/local/lib/python2.7/site-packages/firetv/__main__.py", line 244, in main
    if args.default and not add('default', args.default):
  File "/srv/firetv/firetv/local/lib/python2.7/site-packages/firetv/__main__.py", line 88, in add
    devices[device_id] = FireTV(str(host))
  File "/srv/firetv/firetv/local/lib/python2.7/site-packages/firetv/__init__.py", line 99, in __init__
    self.connect()
  File "/srv/firetv/firetv/local/lib/python2.7/site-packages/firetv/__init__.py", line 109, in connect
    serial=self.host)
TypeError: unbound method ConnectDevice() must be called with AdbCommands instance as first argument (got nothing instead)

Python 2.7 VirtualEnv

Could be due to this version of python-adb? v1.3.0
https://github.com/google/python-adb/releases

Backwards Incompatible changes
adb_commands.AdbCommands() is now a normal class rather than a collection of staticmethods. Using the following example code to get started:

device = adb_commands.AdbCommands()
device.ConnectDevice(rsa_keys=[signer])

**Comment by badgerhome** [comment]
Downgraded to python-adb v1.2.0 and it's now working.

**Comment by chilicheech** [comment]
I'm getting the same error with version 1.3.0:
```
root@35da2cfa8715:/# firetv-server -d 192.168.1.41:5555
Traceback (most recent call last):
  File "/usr/local/bin/firetv-server", line 11, in <module>
    load_entry_point('firetv==1.0.5', 'console_scripts', 'firetv-server')()
  File "/usr/local/lib/python2.7/dist-packages/firetv/__main__.py", line 237, in main
    if args.default and not add('default', args.default):
  File "/usr/local/lib/python2.7/dist-packages/firetv/__main__.py", line 81, in add
    devices[device_id] = FireTV(str(host))
  File "/usr/local/lib/python2.7/dist-packages/firetv/__init__.py", line 61, in __init__
    self.connect()
  File "/usr/local/lib/python2.7/dist-packages/firetv/__init__.py", line 71, in connect
    serial=self.host)
TypeError: unbound method ConnectDevice() must be called with AdbCommands instance as first argument (got nothing instead)
root@35da2cfa8715:/#
```

**Comment by schmiegelt** [comment]
For downgrading, you can insert the following in the Dockerfile, right after the pip upgrade:
`RUN pip --no-cache-dir install adb==1.2.0`

**Comment by effelle** [comment]
Same issue here.
@schmiegelt: how do you do that? Am I wrong or editing the dockerfile means commit the image?
I' ve tryed to edit the image using bash with 

docker run -it --entrypoint=/bin/bash $IMAGE -i
pip --no-cache-dir install adb==1.2.0

but is just temporary, since every time a log out from the image the adb version revert to 1.3.0.

Edit: I' ve successfully added the line on Dockerfile. 
Now the daemon works, but can't connect with my fire Tv:

WARNING:root:Couldn't connect to host: 192.168.15.198:5555, error: Connection refused

Perhaps the server doesn't works with SSL?

**Comment by JeffLIrion** [comment]
This is fixed in 1.0.6.
